doc/source/categorical.rst

```diff
@@ -182,6 +182,19 @@ It's also possible to pass in the categories in a specific order:
     indicate an ordered ``Categorical``.
 
 
+.. note::
+
+    Pay attention! Result of ``.unique()`` not always the same as ``Series.cat.categories``,
```

Comment:
dont' say 'Pay attention!',  you can do a ``::warning::`` if you want, but a note is just fine.

``The results of .unique() are not always the same as .categories.....``